Source/Shaders/Builtin/Functions/cascadeDistance.glsl

```diff
@@ -1,4 +1,7 @@
+
+uniform vec4 u_shadowMapCascadeDistances;
```

Comment:
Search *.glsl, but I _think_ we prefix uniforms like this with `czm_`.  Does it make sense in this context?